blockbuster/priority_nonce.go

```diff
@@ -0,0 +1,477 @@
+package blockbuster
```

Comment:
This is essentially a copy paste from the mempool directory (we should deprecate it in a later PR).